charts/external-es-proxy/templates/external-es-proxy-networkpolicy.yaml

```diff
@@ -0,0 +1,46 @@
+####################################
+## Elasticsearch Proxy NetworkPolicy
+####################################
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "external-es-proxy.fullname" . }}-policy
+  labels:
+    tier: external-logging
+    component: {{ template "external-es-proxy.name" . }}
+    chart: {{ template "external-es-proxy.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  podSelector:
+    matchLabels:
+      tier: external-logging
+      component: {{ template "external-es-proxy.name" . }}
+      release: {{ .Release.Name }}
+  policyTypes:
+  - Ingress
+  ingress:
```

Comment:
Do we know in advance the target ES instance address/host? This is so we could add egress rules as well.